webapp/portlet/src/main/webapp/vue-apps/user-setting-notifications/components/UserSettingNotificationPlugin.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div v-if="isEnabledNotifications.length">
```

Comment:
could you return the length from the computed property as you did in the [UserSettingNotificationGroup.vue](https://github.com/Meeds-io/social/pull/2589/files#diff-d367d001c3f5d3130c3c097edeab782fffb2a8f832840ad33532bc7336dd310d) file